app/views/decidim/initiatives/initiatives/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<%
```

Comment:
Maybe this file can be defaced rather than overriden ?

Reply:
Yes I agree